cypress/integration/token-list/index.spec.ts

```diff
@@ -33,13 +33,28 @@ describe('Token list filter format control and theme control', function () {
     cy.visit('/tokens/list');
   });
 
-  it('controls format of token name', () => {
+  it('controls format of token name (default = css)', () => {
+    const tokenNode = cy.get('[data-cy="tokens-table-container"] li:first dt [data-paste-element="TEXT"]');
+    cy.get('[data-cy="format-control"]').should('have.value', 'css');
```

Comment:
question: does cypress ever keep track of localStorage, or does it use a clean environment each time? in other words, could this test ever fail because 'javascript' was the last value used for the format control in the test, and we pull it from local storage on load? If so, should we do some cleanup after the tests run to clear local storage?

Reply:
I didn't check, but I presume it would clear it. That said, I made the test end on CSS to not have to worry about it.